### ISSUE TITLE
有点问题想请教下

### ISSUE THREAD

**Issue by will-wiki** [bojone/SPACES#25]
苏神您好，看了您项目代码后有点疑惑想请教下：主要是seq2seq_model.py文件里的代码

在处理BIO标签的时候您的代码是 labels = source_labels + target_labels[1:]，但是这个处理的话导致了label的长度比其他几个输入短了一个单位。
def compute_copy_loss(self, inputs, mask=None):
        _, y_mask, y_true, _, y_pred = inputs
        y_mask = K.cumsum(y_mask[:, ::-1], axis=1)[:, ::-1]
        y_mask = K.cast(K.greater(y_mask, 0.5), K.floatx())
        y_mask = y_mask[:, 1:]  # mask标记，减少一位
        y_pred = y_pred[:, :-1]  # 预测序列，错开一位
        loss = K.sparse_categorical_crossentropy(y_true, y_pred)
        loss = K.sum(loss * y_mask) / K.sum(y_mask)
        return loss
y_mask处理后其实是表示全部序列都做loss，但是这里的y_pred和y_true位置是不是对不上了。
比如说：y_pred是在序列上做三分类，labels则是对应的正确类别
y_pred:  cls a b seq cls c d (seq)
labels:   cls a b seq c d seq (bio标签输入)
即使y_pred = y_pred[:, :-1]  # 预测序列，错开一位，长度是对应上了，但是位置好像有问题
